config/routes.rb

```diff
@@ -10,6 +10,8 @@
   # first created -> highest priority.
 
   get "/robots.txt" => RobotsGenerator
+  get "/sitemap.xml.gz" => "sitemap#sitemap", format: :xml
```

Comment:
I don't think you have to.  http://www.sitemaps.org/protocol.html states: "If you would like, you may compress your Sitemap text file using gzip to reduce your bandwidth requirement."  I think you'd just let your users know that they needed to submit the `.gz` url to search engines, and that `sitemap.xml` will 404.

Speaking of that, I also need to add an entry to the robots.txt file with the sitemap name.  Will do that.